mysql-test/suite/rocksdb/include/rpl_row_not_found.inc

```diff
@@ -10,7 +10,8 @@
 
 connection slave;
 call mtr.add_suppression("Slave SQL: Worker.*failed executing transaction");
```

Comment:
Is this one still needed? The one added below looks too similar, is it as 5.6 vs 5.7 difference?